src/posthog-core.ts

```diff
@@ -996,6 +998,7 @@ export class PostHog {
         const startTimestamp = readOnly ? undefined : this.persistence.remove_event_timer(eventName)
         let properties = { ...eventProperties }
         properties['token'] = this.config.token
+        properties['$config_defaults'] = this.config.defaults
```

Comment:
nice

Reply:
nice

(yeah I figure we can use this to warn snippet users about deprecation)